src/coreclr/jit/unwindamd64.cpp

```diff
@@ -399,7 +423,7 @@ void Compiler::unwindSaveRegWindows(regNumber reg, unsigned offset)
             code             = (UNWIND_CODE*)&func->unwindCodes[func->unwindCodeSlot -= sizeof(UNWIND_CODE)];
             code->UnwindOp   = (genIsValidFloatReg(reg)) ? UWOP_SAVE_XMM128_FAR : UWOP_SAVE_NONVOL_FAR;
         }
-        code->OpInfo          = (BYTE)reg;
+        code->OpInfo          = (BYTE)(genIsValidFloatReg(reg) ? reg - XMMBASE : reg);
```

Comment:
need to double check if similar handling is needed for mask registers as well.

Reply:
verified that mask registers are all callee trash so they will never be saved/restored in prolog/epilog.